components/tidb_query_vec_aggr/src/impl_first.rs

```diff
@@ -21,22 +21,26 @@ impl super::AggrDefinitionParser for AggrFnDefinitionParserFirst {
         super::util::check_aggr_exp_supported_one_child(aggr_def)
     }
 
-    fn parse(
+    #[inline]
+    fn parse_rpn(
         &self,
-        mut aggr_def: Expr,
-        ctx: &mut EvalContext,
+        mut root_expr: Expr,
+        exp: RpnExpression,
```

Comment:
Prefer to use the same name, instead of calling `aggr_def` or `exp` sometimes.

Reply:
> Prefer to use the same name, instead of calling `aggr_def` or `exp` sometimes.

Now the original expr-tree is called aggr_def, splitted tree root is called root_expr, rest called exp.